### DISCUSSION TITLE
PHP 7.2 and tidy-html not working

### DISCUSSION THREAD

**Discussion by hellkith** [shivammathur/homebrew-php#496] [Q&A / Help]
Hello,

I am running into an issue with php 7.2 and tidy-html5 v5.8.
Whenever I try to use php 7.2 it gives me the following error:
`dyld: Library not loaded: /usr/local/opt/tidy-html5/lib/libtidy.5.dylib
  Referenced from: /usr/local/Cellar/php@7.2/7.2.34_4/bin/php
  Reason: image not found
`

How can I fix this as the suggestion in https://github.com/shivammathur/homebrew-php/discussions/400 does not work.

**Comment by shivammathur** [comment]
Run `brew update`
Then reinstall tidy-html5
```
brew reinstall tidy-html5
```
